core/01_core.js

```diff
@@ -210,8 +210,35 @@
     return aggregate;
   }
 
+  let reportExceptionCallback = undefined;
+
+  // Used to report errors thrown from functions passed to `queueMicrotask()`.
+  // The callback will be passed the thrown error. For example, you can use this
+  // to dispatch an error event to the global scope.
+  // In other words, set the implementation for
+  // https://html.spec.whatwg.org/multipage/webappapis.html#report-the-exception
+  function setReportExceptionCallback(cb) {
```

Comment:
Another callback, yay :D

Reply:
Indeed. I wonder if this is something to be worried about, and if there wouldn't be alternatives to the ever-growing list of callbacks? :)

Reply:
If you have ideas how it could be done without a callback I'll be glad to explore other ways, but it seems callbacks are often the easiest solution. The problem is that you need to remember to set the callback

Reply:
We can avoid this moving the WHATWG `queueMicrotask()` function out of core, I don't think it belongs there. Should I do that? If needed we can keep a `Deno.core.enqueueMicrotask()` to build it on but the high level function shouldn't be exposed by core IMO.

Reply:
I think the current solution is fine